pytest_mpl/plugin.py

```diff
@@ -331,6 +395,29 @@ def compare_image_to_baseline(self, item, test_image, result_dir):
 
         return compare_images(baseline_image, test_image, tol=tolerance)
 
+    def load_hash_library(self, library_path):
+        with open(library_path) as fp:
+            return json.load(fp)
+
+    def compare_image_to_hash_library(self, item, fig, result_dir):
+        compare = self.get_compare(item)
+
+        # TODO: Should the CLI args or the mark take precedence?
```

Comment:
I copied the precedence of the other keywords, but actually I think it makes more sense for the user to be able to provide an override hash library from the CLI?

Reply:
Yes I agree that CLI should probably take precedence